### PR TITLE
User receives echoed response

### PR DESCRIPTION
**Pull request by lottetreg**
New `Writer` class to handle sending a message to a connection's output stream

### CODE REVIEW

File: src/main/java/io/github/lottetreg/echo/Connection.java [hunk 1]
```diff
@@ -20,6 +21,15 @@ public InputStream getInputStream() {
     }
   }
 
+  public OutputStream getOutputStream() {
+    try {
+      return socket.getOutputStream();
+    } catch(IOException e) {
+      System.out.println(e);
+      return null;
+    }
```
Comment (parkerpersaud) [inline src/main/java/io/github/lottetreg/echo/Connection.java:30]: I commented on the wrong PR earlier - is the exception handling going to come in a future PR?

**Review by parkerpersaud — APPROVED** [review]
I had one (non-blocking) question about the exception handling but otherwise 👍

File: src/main/java/io/github/lottetreg/echo/Connection.java (hunk 1 above)
Comment (lottetreg) [reply src/main/java/io/github/lottetreg/echo/Connection.java:30]: @klkelley yes it is! :)

File: src/test/java/io/github/lottetreg/echo/MockConnection.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
+package io.github.lottetreg.echo;
```
Comment (Thomascountz) [inline src/test/java/io/github/lottetreg/echo/MockConnection.java:1]: I think it's great that you waited to move this out until you needed to! :D

**Review by Thomascountz — APPROVED** [review]
LGTM! 💃

**Pull request merged**